perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -401,7 +401,7 @@ sub ward_check : Private {
         $parent_id = $c->stash->{area}->{id};
     }
 
-    my $qw = FixMyStreet::MapIt::call('area/children', [ $parent_id ],
+    my $qw = $c->cobrand->call_hook('all_reports_ward_areas') || FixMyStreet::MapIt::call('area/children', [ $parent_id ],
```

Comment:
Instead of being a hook, this could be a function in Default.pm that returns the default, and overridden in TfL, passing in ID and cobrand, then could be shared here and in Body.pm location too.

Reply:
Is there a situation where the cobrand on which that function is being called would be different to the cobrand being passed in?